src/explore-education-statistics-common/src/modules/table-tool/components/TableHeadersReadOnlyList.tsx

```diff
@@ -20,14 +20,20 @@ const TableHeadersReadOnlyList = ({
   legend,
   name,
 }: Props) => {
-  const { expandedLists } = useTableHeadersContext();
+  const { expandedLists, reverseOrderIds, isReverse, setIsReverse } =
+    useTableHeadersContext();
   const isExpanded = expandedLists.includes(id);
 
   const { getValues } = useFormContext();
   const list: Filter[] = getValues(name);
 
   const displayItems = isExpanded ? list.length : defaultNumberOfItems;
 
+  if (reverseOrderIds.includes(name) && isReverse) {
+    list.reverse();
```

Comment:
It didn't cause a problem but you are right, creating a copy is better practise. I had to include the reversed copy within the setValue state as well as the saved values were still catching the originals instead of the reversed ones.